yaptide/celery/tasks.py

```diff
@@ -192,3 +192,11 @@ def sh12a_simulation_status(path_to_file: str):
             }
         }
         return sim_info
+
+
+@celery_app.task
+def cancel_simulation(task_id: str) -> dict:
+    """Task responsible for canceling simulation in progress"""
```

Comment:
Is it sending SIGTEM singal to python process and then to the shieldhit executable ?

Reply:
No, it does not

Reply:
What about changing that in separete PR ?